src/test/ui/suggestions/suggest-move-types.stderr

```diff
@@ -133,6 +134,7 @@ LL | struct Dl<'a, 'b, 'c, T, U, V, M: ThreeWithLifetime<T, 'a, A=(), B=(), U, '
    |                                                        ^^
    |
    = note: type arguments must be provided before lifetime arguments
```

Comment:
Wait hold on, shouldn't type arguments be provided _after_ lifetime arguments?

Reply:
Huh. Yes, they should be. But that didn't come from my PR though, so I'm scared to touch it. :laughing:

Reply:
Woah, I'm shocked this wasn't caught before. Thanks for spotting this, @doctorn! I've opened up https://github.com/rust-lang/rust/issues/72815 to make sure we don't lose track of this.